bin/codeclimate-prospector

```diff
@@ -5,40 +5,78 @@ import simplejson as json
 
 config_json = {}
 
-with open('/config.json', 'rb') as fhandle:
-    config_json = json.load(fhandle)
+pylint_codes = {}
 
-config_part = ""
-config_file = config_json.get('config')
-exclude_dirs = config_json.get('exclude_paths')
+remediation_multiplier = {}
 
+severity_types = {
+  'I': 'info',  # info
+  'C': 'normal',  # convention
+  'R': 'normal',  # refactor
+  'W': 'normal',  # warning
+  'E': 'critical',  # error
+  'F': 'critical'  # fatal
+}
 
-if config_file:
-    config_part = "--profile-path /code --profile \"/code/%s\"" % config_file
-else:
-    config_part = "--uses django celery flask --with-tool mccabe --with-tool frosted --with-tool pyroma  --with-tool dodgy --with-tool pyflakes --with-tool pep257 --with-tool pep8 --with-tool pylint --with-tool vulture"
+with open('/usr/src/app/config/pylint-codes.json', 'rb') as fhandle:
```

Comment:
Seems a bit odd that these are at explicit paths.  I would expect them to be overridable from the config file.